tests/components/orderentry/OrderEntryPage.test.jsx

```diff
@@ -57,11 +57,16 @@ describe('Test for Order entry page when orderentryowa.encounterType is set', ()
     mountedComponent = shallow(<OrderEntryPage {...props} />);
     expect(mountedComponent.find('div.error-notice').length).toBe(0);
   });
+  it('renders the patient-orders component', () => {
+    const component =  shallow(<OrderEntryPage {...props} />);
+    component.find('.orders-nav').simulate('click');
+    expect(component.state('currentOrderType')).toEqual({})
```

Comment:
This test should test what is rendered. In the `render()`. One thing to do would be: `wrapper.find` the component you are testing for and expect the `length` to be 1 or whichever. Testing what is in the state might not necessarily mean it is rendered. I hope this makes sense.

Reply:
ohh, that makes sense, I'll look into this